src/logic/Tile.ts

```diff
@@ -1,8 +1,13 @@
-import { Hex } from "honeycomb-grid";
+import { Hex, HexCoordinates } from "honeycomb-grid";
 import ITile from "../types/ITile";
 
 export class Tile extends Hex implements ITile {
-  fill = "";
+  fill: string;
+
+  constructor(coordinates?: HexCoordinates) {
+    super(coordinates);
+    this.fill = "";
```

Comment:
Unrelated to this PR but I think we should change fill to be an enum so it can only be "B" "W" or ""

Reply:
Agreed, good suggestion, let's add that to Jira.